### ISSUE TITLE
LICENSE or NOTICE ???

### ISSUE THREAD

**Issue by seyoung-hyun** [hmkcode/Android#12]
Hi, I'm Seyoung.

I want to use your code in my commercial app.
But I can'f find any terms and conditions for use, reproduction, and distribution.
I wonder if I can use the code for commercial distribution or not.
Please let me know how I can use the code for commercial distribution.

Thanks.

**Comment by pratik7443** [comment]
Which code are you speaking about ,
Can u pass on the link

**Comment by JSBACH100** [comment]
Yes, please let me know which repository or code you are referring to. Thank you.

**Comment by seyoung-hyun** [comment]
@hmkcode @pratik7443 @JSBACH100 

I want to use this code :)

https://github.com/hmkcode/Android/blob/master/android-show-image-and-path/src/com/hmkcode/android/image/RealPathUtil.java

**Comment by pratik7443** [comment]
Yes, you can .

Regards,
*Pratik V*
*pratik7443@gmail.com <pratik7443@gmail.com>*
*Software Developer (Android)*
